packages/ui/src/components/shared/settings/ManageDappAccessDialog/DappAccessItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { AppInfo } from '@coong/base/requests/WalletState';
+import { useWalletState } from 'providers/WalletStateProvider';
+
+interface DappAccessItemProps {
+  appInfo: AppInfo;
+}
+export default function DappAccessItem({ appInfo }: DappAccessItemProps): JSX.Element {
+  const { walletState } = useWalletState();
+  const { id, name, url, authorizedAccounts } = appInfo;
+  const appId = id || walletState.extractAppId(url);
+
+  return (
+    <div className='border border-black/10 dark:border-white/15 py-2 px-4 rounded flex flex-row items-center gap-4 bg-zinc-50 dark:bg-black/15'>
```

Comment:
```suggestion
    <div className='border border-black/10 dark:border-white/15 py-2 px-4 rounded flex items-center gap-4 bg-zinc-50 dark:bg-black/15'>
```
I think we don't need to use `flex-row` here because it's the default when using `flex`. Please check another spot.